runtime/compiler/control/CompilationThread.cpp

```diff
@@ -3116,8 +3116,13 @@ void TR::CompilationInfo::stopCompilationThreads()
 
       fprintf(stderr, "Number of compilations per level:\n");
       for (int i = 0; i < (int)numHotnessLevels; i++)
+         {	      
          if (_statsOptLevels[i] > 0)
-            fprintf(stderr, "Level=%d\tnumComp=%d\n", i, _statsOptLevels[i]);
+            {		 
+            fprintf(stderr, "Level=%d\tnumComp=%d\tnumRemoteComp=%d\n", i, _statsOptLevels[i], _statsRemoteOptLevels[i]);
+            JITServerHelpers::printJITServerMsgStats(jitConfig);
```

Comment:
This code prints the same message stats a few times (for each opt level). Here we want stats about the remote compilations, not about messages.

Reply:
Also, please change `JITServerHelpers::printJITServerMsgStats();` to display total number of messages and average number of messages per compilation.